builtin/game/chatcommands.lua

```diff
@@ -103,6 +103,30 @@ core.register_chatcommand("privs", {
 	end,
 })
 
+core.register_chatcommand("hasprivs", {
+	params = "<priv>",
+	description = "Returns a list of all online players with the priv passed as param. e.g. /hasprivs fly",
```

Comment:
Should be "Return" not "Returns" for consistency with other descriptions. This is also a pretty long and overly technical description.
Shorter: "Return list of all online players with privilege."
If you want to include an example, the only other example in a description is in parenthesis, so it should stay consistent.

Reply:
^ Agreed.

Reply:
@paramat That's already been fixed in [b91f8a](https://github.com/minetest/minetest/pull/7336/commits/b91f8a4a2b922aa9bc99d5b0f89e52d45d3b196e)